underscore.js

```diff
@@ -1100,6 +1100,9 @@
 
    // Return a copy of the object without the blacklisted properties.
   _.omit = restArgs(function(obj, keys) {
+    if(obj != null && Object.getOwnPropertySymbols(obj) != null){
```

Comment:
> Regardless, FF and Chrome both added support only about a year ago, which is typically less than our support window.

The browsers with symbol support have the method so there's not an issue of inconsistent support. If they support symbols then it'll work. Yes they'll need to be method existence checks.